spec/octokit/client/repositories_spec.rb

```diff
@@ -273,6 +273,26 @@
     end
   end # .collaborators
 
+  describe ".topics", :vcr do
+    it "returns repository topics" do
+      topics = Octokit.topics("github/linguist", :accept => 'application/vnd.github.mercy-preview+json')
```

Comment:
Instead of using the string, could we use the value from the `PREVIEW_TYPES` hash?
https://github.com/octokit/octokit.rb/blob/master/lib/octokit/preview.rb#L19